Revert-tvOS/Configurations/Revert-tvOS-Release.xcconfig

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
I think most of the project configuration in these xcconfig files could be shared between the iOS and tvOS targets.  We should extract the shared configuration and `#include` that into each of the target specific configs.